applications/Unity.GrantManager/src/Unity.GrantManager.Domain.Shared/GrantApplications/ProjectInfoOptionsList.cs

```diff
@@ -27,4 +27,21 @@ public static class ProjectInfoOptionsList
             new KeyValuePair<string, string>("NO", "No"),
       });
 
+     public static ImmutableDictionary<string, string> OrganizationTypeList =>
+     ImmutableDictionary.CreateRange(new[]
+       {
+              new KeyValuePair<string, string>("CORPORATION", "Corporation"),
+              new KeyValuePair<string, string>("PARTNERSHIP", "Partnership"),
+              new KeyValuePair<string, string>("INCORPORATED_COOPERATIVE", "Incorporated Cooperative"),
+              new KeyValuePair<string, string>("FIRSR_NATION", "First Nation or Indigenous-Owned Enterprises"),
```

Comment:
spelling FIRST_NATION ?

Reply:
> spelling FIRST_NATION ?

@AndreGAot  Corrected the spelling mistake.